templates/patroni.yml.j2

```diff
@@ -59,6 +59,7 @@ bootstrap:
     retry_timeout: 10
     maximum_lag_on_failover: 1048576
     synchronous_mode: true
+    synchronous_mode_strict: false
```

Comment:
Start as false to let the cluster bootstrap. Since this will be updated in the DCS as soon as the cluster is running, should we have it in the yaml file?

Reply:
I think it's not needed, as you commented. But I don't see any problem with keeping it here either, so you can keep it.

Reply:
No strict positioning here.